profiling/report.py

```diff
@@ -15,7 +15,16 @@
 parser.add_argument("--event", help="Github event name", default="none")
 parser.add_argument("--prof-dir", help="Profiling results directory", default="prof")
 parser.add_argument(
-    "--max-benchmark", help="Maximum number of benchmarks to plot", type=int, default=5
+    "--no-save", help="Do not save the profiling results", action="store_true"
+)
+parser.add_argument(
+    "--no-plot", help="Do not plot the profiling results", action="store_true"
+)
+parser.add_argument(
+    "--selector", help="Filter the filenames by regex, e.g. 'splash', 'pmodel'"
+)
+parser.add_argument(
+    "--max-benchmark", help="Max number of benchmarks to plot", type=int, default=5
```

Comment:
I agree - I will convert these changes to a new PR with more comments. I also think it would be cleaner to convert the code into a script with these arguments.